.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "backward-cpp"]
 	path = backward-cpp
 	url = https://github.com/bombela/backward-cpp.git
+[submodule "libpfm4"]
+	path = libpfm4
+	url = https://git.code.sf.net/p/perfmon2/libpfm4
```

Comment:
As far as you know, this is the git view of the official `libpfm4` repository which lives on sourceforce?

Reply:
My comment here raced with your comment, it seems this answer is "Yes".

Reply:
Yep. [Perfmon2's web site](http://perfmon2.sourceforge.net/) links to [right to it](https://sourceforge.net/p/perfmon2/libpfm4/ci/master/tree/).